react-backend/shared/controllers/feedback.js

```diff
@@ -1,24 +1,55 @@
-const cbAdmin = require('../models/cb-admin');
+const cbAdmin = require('../models/cb_admin');
 
-module.exports = (req, res, next) => {
-  const pgPool = req.app.get('client:psql');
+const createDate = str => {
```

Comment:
As I said in [my comment](https://github.com/TwinePlatform/DataPower/pull/281#discussion_r176271323), this is unnecessary.

Reply:
I wanted to return a `400` if incorrect data was sent, so this was my attempt at type safing the query. In my current layout any postgres error is being caught and passed to middleware returning a `500`.

Reply:
Good point. Would be caught by Joi if you use the validation middleware though :wink:

Reply:
🙈 it's on my to do list...